testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -760,4 +768,23 @@ public void testRowLevelDelete()
         assertQuery("SELECT count(*) FROM " + tableName, "VALUES 24");
         assertUpdate("DROP TABLE " + tableName);
     }
+
+    protected Consumer<Plan> assertPartialLimitWithPreSortedInputsCount(Session session, int expectedCount)
```

Comment:
Not sure this is generally better here, but it's the only common ancestor for the Hive and Phoenix tests.